<h3 id="CollectRawWgsMetrics">CollectRawWgsMetrics</h3>
<section>
<p>Collect whole genome sequencing-related metrics.  This tool computes metrics that are useful for evaluating coverage and performance of whole genome sequencing experiments. These metrics include the percentages of reads that pass minimal base- and mapping- quality filters as well as coverage (read-depth) levels. <br /><br />  The histogram output is optional and for a given run, displays two separate outputs on the y-axis while using a single set of values for the x-axis.  Specifically, the first column in the histogram table (x-axis) is labeled "coverage" and represents different possible coverage depths.  However, it also represents the range of values for the base quality scores and thus should probably be labeled "sequence depth and base quality scores". The second and third columns (y-axes) correspond to the numbers of bases at a specific sequence depth "count" and the numbers of bases at a particular base quality score "baseq_count" respectively.<br /><br />Although similar to the CollectWgsMetrics tool, the default thresholds for CollectRawWgsMetrics are less stringent.  For example, the CollectRawWgsMetrics have base and mapping quality score thresholds set to "3" and "0" respectively, while the CollectWgsMetrics tool has the default threshold values set to "20" (at time of writing).  Nevertheless, both tools enable the user to input specific threshold values.<h4>Usage example:</h4><pre>java -jar picard.jar CollectRawWgsMetrics \<br />      I=input.bam \<br />      O=raw_wgs_metrics.txt \<br />      R=reference_sequence.fasta \<br />      INCLUDE_BQ_HISTOGRAM=true</pre><hr />Please see <a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#CollectWgsMetrics.WgsMetrics'>the WgsMetrics documentation</a> for detailed explanations of the output metrics.<hr /></p>
<table>
<tr><th>Option</th><th>Description</th></tr>
<tr><td>MINIMUM_MAPPING_QUALITY (Integer)</td><td>Minimum mapping quality for a read to contribute coverage.  Default value: 0. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>MINIMUM_BASE_QUALITY (Integer)</td><td>Minimum base quality for a base to contribute coverage.  Default value: 3. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>COVERAGE_CAP (Integer)</td><td>Treat bases with coverage exceeding this value as if they had coverage at this value.  Default value: 100000. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>LOCUS_ACCUMULATION_CAP (Integer)</td><td>At positions with coverage exceeding this value, completely ignore reads that accumulate beyond this value (so that they will not be considered for PCT_EXC_CAPPED).  Used to keep memory consumption in check, but could create bias if set too low  Default value: 200000. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>INPUT (File)</td><td>Input SAM or BAM file.  Required. </td></tr>
<tr><td>OUTPUT (File)</td><td>Output metrics file.  Required. </td></tr>
<tr><td>REFERENCE_SEQUENCE (File)</td><td>The reference sequence fasta aligned to.  Required. </td></tr>
<tr><td>STOP_AFTER (Long)</td><td>For debugging purposes, stop after processing this many genomic bases.  Default value: -1. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>INCLUDE_BQ_HISTOGRAM (Boolean)</td><td>Determines whether to include the base quality histogram in the metrics file.  Default value: false. This option can be set to 'null' to clear the default value. Possible values: {true, false} </td></tr>
<tr><td>COUNT_UNPAIRED (Boolean)</td><td>If true, count unpaired reads, and paired reads with one end unmapped  Default value: false. This option can be set to 'null' to clear the default value. Possible values: {true, false} </td></tr>
<tr><td>SAMPLE_SIZE (Integer)</td><td>Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.  Default value: 10000. This option can be set to 'null' to clear the default value. </td></tr>
</table>
</section>
