<h3 id="CollectRrbsMetrics">CollectRrbsMetrics</h3>
<section>
<p>Collect metrics from reduced representation bisulfite sequencing (RRBS) data.  This tool collect metrics for RRBS data, based on the methylation status of cytosine (C) bases in both CpG and non-CpG sites across all reads of a BAM/SAM file. For a brief primer on bisulfite sequencing and cytosine methylation, see the <a href='https://www.broadinstitute.org/gatk/guide/article?id=6330'>GATK Dictionary</a>.<br /><br />Since cytosine methylation is not exclusive for CpG "hotspots", the CollectRrbsMetrics tool outputs a summary table indicating the number of CpG and non-CpG cytosines as well as their conversion C -> T (+ strand) or G -> A (- strand) rates. The tool also outputs the numbers of reads having no CpG sites, and the numbers of reads discarded from the analysis due to inadequate size or excessive numbers of mismatches.<br /><br />The tool also provides a table containing detailed information on CpG occurrence frequency, CpG conversion frequencies [C -> T (+ strand) or G -> A (- strand)], and the specific locations of the CpG sites in the genome. The conversion frequency helps determines the methylation status of a CpG site.<br /><br />Finally, the tool provides graphical representation of four metrics in the form of a ".pdf" document. These metrics are the bisulfite conversion rate for CpG and non-CpG cytosines, a distribution of the numbers of CpG sites as a function of CpG conversion rate, the distribution of CpG sites by read coverage, and the numbers of reads discarded due to high numbers of mismatches or inadequate read size.<h4>Usage example:</h4><pre>java -jar picard.jar CollectRrbsMetrics \<br />      I=input.bam \<br />      M=rrbs_metrics \<br />      R=reference_sequence.fasta</pre><hr />Please see <a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#RrbsCpgDetailMetrics'>the RrbsCpgDetailMetrics documentation</a> and the <a href='https://broadinstitute.github.io/picard/picard-metric-definitions.html#RrbsSummaryMetrics'>the RrbsSummaryMetrics documentation</a>for detailed explanations of the output metrics.<hr /></p>
<table>
<tr><th>Option</th><th>Description</th></tr>
<tr><td>INPUT (File)</td><td>The BAM or SAM file containing aligned reads. Must be coordinate sorted  Required. </td></tr>
<tr><td>METRICS_FILE_PREFIX (String)</td><td>Base name for output files  Required. </td></tr>
<tr><td>REFERENCE (File)</td><td>The reference sequence fasta file  Required. </td></tr>
<tr><td>MINIMUM_READ_LENGTH (Integer)</td><td>Minimum read length  Default value: 5. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>C_QUALITY_THRESHOLD (Integer)</td><td>Threshold for base quality of a C base before it is considered  Default value: 20. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>NEXT_BASE_QUALITY_THRESHOLD (Integer)</td><td>Threshold for quality of a base next to a C before the C base is considered  Default value: 10. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>MAX_MISMATCH_RATE (Double)</td><td>Maximum percentage of mismatches in a read for it to be considered, with a range of 0-1  Default value: 0.1. This option can be set to 'null' to clear the default value. </td></tr>
<tr><td>SEQUENCE_NAMES (String)</td><td>Set of sequence names to consider, if not specified all sequences will be used  Default value: null. This option may be specified 0 or more times. </td></tr>
<tr><td>ASSUME_SORTED (Boolean)</td><td>If true, assume that the input file is coordinate sorted even if the header says otherwise.  Default value: false. This option can be set to 'null' to clear the default value. Possible values: {true, false} </td></tr>
<tr><td>METRIC_ACCUMULATION_LEVEL (MetricAccumulationLevel)</td><td>The level(s) at which to accumulate metrics.    Default value: [ALL_READS]. This option can be set to 'null' to clear the default value. Possible values: {ALL_READS, SAMPLE, LIBRARY, READ_GROUP} This option may be specified 0 or more times. This option can be set to 'null' to clear the default list. </td></tr>
</table>
</section>
